### PR TITLE
chore: prepare themes for v1/v2 split

### PR DESCRIPTION
**Pull request by kazo0**
## PR Type

- Other... Please describe:

Prepare for Themes v2, starting with implementing Material Design 3

### CODE REVIEW

File: Uno.Themes.Common/Uno.Themes.Common.projitems [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (Xiaoy312) [inline Uno.Themes.Common/Uno.Themes.Common.projitems:1]: `Uno.Themes.Common\` (from root directory) should be removed

File: src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+                    xmlns:um="using:Uno.Material"
+                    mc:Ignorable="ios macos">
+
+    <x:String x:Key="M3MaterialRevealGlyphPathData">M16,5.5999984C17.699951,5.5999984 19.099976,6.9999918 19.099976,8.6999893 19.099976,10.400001 17.699951,11.799994 16,11.799994 14.299988,11.799994 12.899963,10.400001 12.899963,8.6999893 12.899963,6.9999918 14.299988,5.5999984 16,5.5999984z M16,2.1999893C12.399963,2.1999893 9.5,5.0999984 9.5,8.6999893 9.5,12.299994 12.399963,15.199988 16,15.199988 19.599976,15.199988 22.5,12.299994 22.5,8.6999893 22.5,5.0999984 19.599976,2.1999893 16,2.1999893z M16,0C24.799988,5.2249334E-08 32,8.6999893 32,8.6999893 32,8.6999893 24.799988,17.400001 16,17.400001 7.1999512,17.400001 0,8.6999893 0,8.6999893 0,8.6999893 7.1999512,5.2249334E-08 16,0z</x:String>
+
+    <SolidColorBrush x:Key="M3MaterialDisabledOutlinedPasswordBoxBorderBrush"
+                     Opacity="0.12"
+                     Color="{ThemeResource OnSurfaceColor}" />
+
+    <Style x:Name="M3MaterialRevealButtonStyle"
```
Comment (Xiaoy312) [inline src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml:16]: x:
```suggestion
    <Style x:Key="M3MaterialRevealButtonStyle"
```

Comment (kazo0) [reply src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml:16]: Nice catch, the v1 PasswordBox styles also have this issue.. gonna do a pass to see if there are others

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/TextBoxSamplePage.xaml [hunk 3]
```diff
@@ -102,6 +102,86 @@
...
+						<!--<smtx:XamlDisplay UniqueKey="M3Material_TextBoxSamplePage_4"
+										  Style="{StaticResource XamlDisplayBelowStyle}">
+							<TextBox Style="{StaticResource M3MaterialOutlinedTextBoxStyle}" />
+						</smtx:XamlDisplay>-->
+
+						<!-- TextBox Outlined with PlaceholderText -->
+						<!--<smtx:XamlDisplay UniqueKey="M3Material_TextBoxSamplePage_5"
+										  Style="{StaticResource XamlDisplayBelowStyle}">
+							<TextBox PlaceholderText="Outlined"
+									 Style="{StaticResource M3MaterialOutlinedTextBoxStyle}" />
```
Comment (agneszitte-nventive) [inline src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/TextBoxSamplePage.xaml:140]: @kazo0 did you find what's not working for the M3MaterialOutlinedTextBoxStyle for UWP at the end ?
(as these samples are still commented)

**Review by agneszitte-nventive — COMMENTED** [review]
@kazo0 just to let you know that with the latest webapp version there is no selected index by default for the ComboBox (it's empty). In case you didn't notice it ;)

**Review by agneszitte-nventive — APPROVED** [review]

**Pull request merged**